### DISCUSSION TITLE
How to add Photoview to Nginx Proxy Manager

### DISCUSSION THREAD

**Discussion by PrzemekSkw** [photoview/photoview#736] [General]
Hello, my Photoview works great in LAN network but now I want to add it as subfolder to my Nging Proxy Manager with https domain. How to add base url to docker-compose to make it work?

Regards,
Przemek

**Comment by viktorstrate** [comment]
That is not possible from Photoview, but you can probably configure that with Nginx Proxy Manager directly https://medium.com/rahasak/nginx-reverse-proxy-with-url-rewrite-a3361a35623c

**Comment by PrzemekSkw** [comment]
Thanks for your answer @viktorstrate but I use Nginx Proxy Manager because I don't understand that coding 😃 and don't know what to do with that.
Regards.

**Reply by sloniupl** [reply]
I just simply added it to Nginx Proxy Manager after specyfying what port it should run in docer-compose - on mine port 8086.
![image](https://user-images.githubusercontent.com/5970551/200263442-e92eace1-a91a-401a-9c16-e4a95a2767a1.png)
SSL you are adding as for any other one in Nginx Proxy Manager.

**Reply by PrzemekSkw** [reply]
> I just simply added it to Nginx Proxy Manager after specyfying what port it should run in docer-compose - on mine port 8086. ![image](https://user-images.githubusercontent.com/5970551/200263442-e92eace1-a91a-401a-9c16-e4a95a2767a1.png) SSL you are adding as for any other one in Nginx Proxy Manager.

OK thanks, it works.
